src/components/share-structure/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/share-structure",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
Don't change this. Lerna will do it for us when we publish this component (step 14 in readme).

Do you think the readme should be updated to avoid this mistake? You could do it in this PR.

Reply:
Ya I will update the Readme